lib/view_component/rails/tasks/view_component.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+task stats: "view_component:statsetup"
+
+namespace :view_component do
+  task statsetup: :environment do
+    require "rails/code_statistics"
+
+    ::STATS_DIRECTORIES << ["Components", Rails.application.config.view_component.view_component_path]
```

Comment:
What do you think about:

```suggestion
    ::STATS_DIRECTORIES << ["ViewComponents", Rails.application.config.view_component.view_component_path]
```